development-docs/DEV_GUIDE.md

```diff
@@ -34,15 +50,239 @@ structure after everything is said and done should look like `/Library/Java/Java
+
+    make build
+
+This labels your latest container build as `strimzi/access-operator:latest` and you can then deploy the standard deployment without
+changing the image target. However, this will only work if all instances of the `imagePullPolicy:` setting are set
+to `IfNotPresent` or `Never`. If not, then the cluster nodes will go to the upstream registry (Quay by default)
+and pull the official images instead of using your freshly built image.
+
+## DCO Signoff
+
+The project requires that all commits are signed-off, indicating that _you_ certify the changes with the developer
+certificate of origin (DCO) (https://developercertificate.org/). This can be done using `git commit -s` for each commit
+in your pull request. Alternatively, to signoff a bunch of commits you can use `git rebase --signoff _your-branch_`.
+
+You can add a commit-msg hook to warn you if the commit you just made locally has not been signed off. Add the following
+line to you `.git/hooks/commit-msg` script to print the warning:
+
+```
+./tools/git-hooks/signoff-warning-commit-msg $1
+```
+
+## Checkstyle pre-commit hook
+
+The Checkstyle plugin runs on all pull requests to the Strimzi repository. If you haven't compiled the code via maven,
+before you submit the PR, then formatting bugs can slip through and this can lead to annoying extra pushes to fix
+things. In the first instance you should see if your IDE has a Checkstyle plugin that can highlight errors in-line, such
+as [this one](https://plugins.jetbrains.com/plugin/1065-checkstyle-idea) for IntelliJ.
+
+You can also run the Checkstyle plugin for every commit you make by adding a pre-commit hook to your local Strimzi git
+repository. To do this, add the following line to your `.git/hooks/pre-commit` script, to execute the checks and fail
+the commit if errors are detected:
 
-To build the Docker image:
+```
+./tools/git-hooks/checkstyle-pre-commit
+```
 
-    make all
+## Building container images for other platforms with Docker `buildx`
 
-Once this completes you should have a new image called `strimzi/access-operator:latest`
+Docker supports building images for different platforms using the `docker buildx` command. If you want to use it to
+build the ioeratir unage, you can just set the environment variable `DOCKER_BUILDX` to `buildx`, set the environment
+variable `DOCKER_BUILD_ARGS` to pass additional build options such as the platform and run the build. For example
+following can be used to build the image for Linux on Arm64 / AArch64:
 
-The tests run during the build can be skipped by setting the `MVN_ARGS` environment variable and passing that to the
-make command:
+```
+export DOCKER_BUILDX=buildx
+export DOCKER_BUILD_ARGS="--platform linux/amd64 --load"
+make all
+```
 
-    make MVN_ARGS='-DskipTests' all
+_Note: Strimzi currently does not officially support any other platforms then Linux on `amd64`._
```

Comment:
Should this say _Strimzi Access operator_? Since Strimzi in general supports Aarch64/arm64 and s390x`.

Reply:
Yes I'll update it